docs/theme.js

```diff
@@ -112,6 +112,29 @@ export default {
       p: 3,
       borderColor: 'border',
       borderRadius: 2,
-    }
-  }
+    },
+  },
+  dropdownMenu: {
+    primary: {
+      dropdownTrigger: {
+        border: '1px solid #ddd',
```

Comment:
- [ ] Don't hard code the `borderWidth` and `borderColor`. Those value should be referred from theme.